test/unit/configuration_test.rb

```diff
@@ -27,28 +29,27 @@ class ConfigurationTest < Minitest::Test
         "custom_associations" => ["custom_association"],
         "inflections_file" => "custom_inflections.yml",
       }
-      YAML.expects(:load_file).with("./packwerk.yml").returns(configuration_hash)
 
-      configuration = Configuration.from_path(".")
+      merge_into_yaml_file("packwerk.yml", configuration_hash)
 
-      assert_equal ["xyz/*.rb"], configuration.include
-      assert_equal ["{exclude_dir,bin,tmp}/**/*"], configuration.exclude
-      assert_equal File.expand_path("."), configuration.root_path
```

Comment:
This set configuration.root_path to `Shopify/packwerk`, and not `Shopify/packwerk/test/fixtures/skeleton`